.babelrc

```diff
@@ -1,17 +1,21 @@
 {
-  "presets": [
-    ["@babel/env",
-      {
-        "targets": {
-          "node": "10"
-        },
-        "debug": false,
-        "useBuiltIns": "usage",
-        "corejs": 3,
-        "shippedProposals": true
-      }
-    ],
-    "@babel/preset-typescript"
-  ],
-  "plugins": ["@babel/plugin-transform-runtime"]
+	"presets": [
+		[
+			"@babel/env",
+			{
+				"targets": {
+					"node": "10"
+				},
+				"debug": false,
+				"useBuiltIns": "usage",
+				"corejs": 3,
+				"shippedProposals": true
+			}
+		],
+		"@babel/preset-typescript"
+	],
+	"plugins": [
```

Comment:
@raiseandfall what's up with formatting here?

Reply:
Ah not sure. Maybe my JSON formatting is set differently on my editor.